### PR TITLE
adding Soumya Banerjee projects

### PR DESCRIPTION
**Pull request by neelsoumya**
Adding some project ideas for joint supervision

### CODE REVIEW

File: _availableprojects/xAI_unconventional_AI.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+2. Rudin C (2019) Stop explaining black box machine learning models for high stakes decisions and use interpretable models instead. Nat Mach Intell 1: 206–215. 
+
+3. Banerjee S, tom rp Bishop (2022) dsSynthetic: Synthetic data generation for the DataSHIELD federated analysis system. BMC Res Notes 15: 230. 
+
+4. Banerjee S, Sofack GN, Papakonstantinou T, Avraam D, Burton P, et al. (2022) dsSurvival: Privacy preserving survival models for federated individual patient meta-analysis in DataSHIELD. BMC Res Notes 15: 197.
+
+supervisors:
+  - Soumya Banerjee
+  - lawrennd
```
Comment (apaleyes) [inline _availableprojects/xAI_unconventional_AI.md:69]: These need to match handles here:
https://github.com/mlatcl/mlatcl.github.io/tree/main/_people

thus:
```
 - soumya-banerjee
 - neil-d-lawrence
```

**Review by apaleyes — COMMENTED** [review]
Looks fine, except for one comment below.

File: _availableprojects/xAI_unconventional_AI.md (hunk 1 above)
Comment (neelsoumya) [reply _availableprojects/xAI_unconventional_AI.md:69]: thanks!

Comment (neelsoumya) [reply _availableprojects/xAI_unconventional_AI.md:69]: Thanks @apaleyes I closed this PR, edited the file and sent you another PR

**Pull request closed without merging**